doc/cephfs/cephfs-mirroring.rst

```diff
@@ -398,6 +398,69 @@ created, the mirror daemon will clear the failed state upon successful synchroni
 When mirroring is disabled, the respective `fs mirror status` command for the file system
 will not show up in command help.
 
+Metrics
+-------
+
+CephFS exports mirroring metrics as :ref:`Labeled Perf Counters` which will be consumed by the OCP/ODF Dashboard to provide monitoring of Geo Replication in the OCP and ACM dashboard and elsewhere. This would generate the progress of cephfs_mirror syncing and thus provide the monitoring capability. CephFS exports the following mirroring metrics, which could be displayed using the ``counter dump`` command.
```

Comment:
Reword in present tense please.  And give some context for what these OCP and ACM dashboards are.

Reply:
I don't remember from where I got (copy-pasted) that text. Updated the paragraph and dropped over-mentioning about the users. Please check now.